taskpools/chase_lev_deques.nim

```diff
@@ -48,9 +48,7 @@ type
     ## Backend buffer of a ChaseLevDeque
     ## `capacity` MUST be a power of 2
 
-    # Note: update tp_allocUnchecked allocation if any field changes.
-    # Unused. There is no memory reclamation scheme.
-    prev: ptr Buf[T]
+    prevRetired: ptr Buf[T] # intrusive linked list. Used for garbage collection
```

Comment:
Maybe `prevGarbage`, or even just `garbage` to keep nomenclature consistent, instead of using `retired` and `garbage` for the same concept.

also not sure about `intrusive` in comment.